components/count-badge/demo/count-badge.html

```diff
@@ -41,6 +41,19 @@ <h2>Large Count Badge</h2>
 				</template>
 			</d2l-demo-snippet>
 
+			<h2>Override Colorway of Count Badge</h2>
+			<d2l-demo-snippet>
+				<template>
+					<d2l-count-badge id="colorway-override" size="large" text="1 new notification." type="count" number="1"></d2l-count-badge>
+
+					<script>
```

Comment:
I think this would also work:
```html
<d2l-count-badge style="--d2l-count-badge-background-color: purple; --d2l-count-badge-foreground-color: white;" ...></d2l-count-badge>
```

Reply:
This can definitely also work and was the original way that I had implimented it but it caused issues with stylelint that I couldn't get around without an igore statement or file bypass being added in `.stylelintrc.json`.
I'm fine to revert it back to that solution as the demo was cleaner / more intuitive but this way relays the same information without causing stylelint issue

Reply:
I see. I'd probably just add an ignore statement, but I'm ok either way. I probably wouldn't add a one-off file exception to `.stylelintrc.json` though.

Reply:
Stylelint doesn't seem to be playing nicely with parsing html inline styles when it comes to rules exceptions. 

I've swapped the current example to at least make use of a <style> tag rather than js style override.
https://github.com/BrightspaceUI/core/pull/4695/commits/035d517f2060ba8393346f28797991a831c7dc9d